plugins/utils/tmux-navigator.nix

```diff
@@ -6,6 +6,43 @@
   ...
 }:
 with lib; let
+  # All `:TmuxNavigate*` commands
+  commands = [
+    "left"
+    "down"
+    "up"
+    "right"
+    "previous"
+  ];
+
+  # Define a submodule so we get docs and enum type-safety for `action`
+  keymapOptions = {
+    # FIXME: Do we really want _all_ of `mapConfigOptions` here?
+    #    Should we consider filtering it? Etc?
+    options =
+      (recursiveUpdate helpers.keymaps.mapConfigOptions (mapAttrs (n: v: {${n}.default = v;}) defaultOptions))
```

Comment:
Thoughts on this? I can remove the `FIXME` if this approach is ok. Otherwise, how would you set-up the submodule options?